subprojects/emcee/queue-api-models/src/main/kotlin/com/avito/emcee/queue/BucketResult.kt

```diff
@@ -0,0 +1,47 @@
+package com.avito.emcee.queue
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+public data class BucketResult(
+    @Json(name = "testDestination")
+    val device: DeviceConfiguration,
+    val unfilteredResults: List<UnfilteredResult>
+) {
+    @JsonClass(generateAdapter = true)
+    public data class UnfilteredResult(
+        val testEntry: TestEntry,
+        val testRunResults: List<TestRunResult>
+    ) {
+        @JsonClass(generateAdapter = true)
+        public data class TestRunResult(
+            val udid: String,
+            val duration: Int, // in seconds
```

Comment:
```suggestion
            val durationSec: Int,
```
OR
```suggestion
            val duration: Duration,
```
It's better to delegate type checking to the compiler.